index.css

```diff
@@ -69,3 +69,216 @@
+  color: #1c1a19;
+}
+
+.works {
+  display: grid;
+}
+
+.storie h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.storie p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.tools {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+.tools li {
+  list-style: none;
+  padding: 10px;
+  margin-right: 10px;
+  border: 1px solid black;
+}
+
+.button {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  text-align: center;
+  background-color: #ff6b00;
+  color: #fff;
+  border: 2px solid #ff6b00;
+}
+
+.button:hover {
+  color: #ff6b00;
+  background-color: #fff;
+  border: 2px solid #ff6b00;
+}
+
+.button:active {
+  color: #fff;
+  background-color: #ff6b00;
+}
+
+.button:disabled {
+  background-color: #5b5957;
+}
+
+.art {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-image: url("images/art.png");
+  width: 327px;
+  height: 386px;
+}
+
+.art h2 {
+  width: 295px;
+  height: 81;
+  margin-top: 8px;
+  margin-left: 16px;
+  font-family: 'crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.art p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.tools1 {
+  display: flex;
+}
+
+.tools1 li {
+  list-style: none;
+  color: #fff;
+  border: 1px solid #fff;
+  margin-right: 5px;
+  background: #ffffff3d;
+  padding: 10px;
+  text-align: center;
+}
+
+article {
+  margin-top: 10px;
+}
+
+.aboutme {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  margin-top: 20px;
+  width: 100%;
+  height: 1901px;
+  background-color: #1c1a19;
+}
+
+.aboutme h2 {
+  font-family: 'Crete Round', sans-serif;
+  width: 327px;
+  height: 79px;
+  font-weight: 400;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  text-align: center;
+}
+
+.aboutme p {
+  height: 148px;
+  margin-top: 20px;
+  margin-left: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.lang {
+  color: #fff;
+}
+
+.lang li {
+  list-style: none;
+}
+
+.lang > ul li {
+  width: 164px;
+  height: 24px;
+}
+
+.lang img {
+  width: 18px;
+  height: 18px;
+  margin-right: 10px;
+}
+
+.lang li ul li {
+  font-family: 'Inter', sans-serif;
+  width: 162px;
+  height: 24;
+  font-size: 22px;
+  line-height: 42.67px;
+  text-align: center;
+  font-weight: 400;
+}
+
+.lang > li:first-child {
+  margin-left: 12px;
+}
+
+.lang > li:last-child {
+  margin-left: 42px;
+}
+
+.lang li > span {
+  width: 164px;
+  height: 24px;
+  font-size: 32px;
+  font-weight: 700;
+  font-family: 'InaiMathi', sans-serif;
+}
```

Comment:
- [ ] Please use the correct margins and alignment in the about me section so it resembles the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2154), below on the right is the Figma design and on the left is your design, kindly fix your design 😁😁

<div>
  <img src="https://user-images.githubusercontent.com/20567503/176933963-79e4afac-3645-4742-9188-8ad5d03e4f78.png" width="300" />
  <img src="https://user-images.githubusercontent.com/20567503/176934146-fa525731-ac04-4150-a211-393a3ba25d7d.png" width="300" />
</div>

Reply:
This issue is well resolved! ✔️